components/GlobalSpinner/GlobalSpinner.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import React from 'react';
+import Image from 'next/image';
+
+/**
+ * GlobalSpinner component
+ * @returns {React.JSX.Element} GlobalSpinner component
+ */
+const GlobalSpinner = (): React.JSX.Element => (
+  <div className="global-spinner w-full h-full grid place-content-center">
+    <Image
+      alt="Gridiron Survivor logo fading in and out"
+      src="../../assets/global-spinner.svg"
```

Comment:
Write a unit test that this component renders and check that this file is in the src. If we update it, we'll know that our test should be updated later too.

Reply:
Added in latest push